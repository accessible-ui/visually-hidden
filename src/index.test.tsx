import * as React from 'react'
import {render} from '@testing-library/react'
import {VisuallyHidden} from './index'

describe('<VisuallyHidden>', () => {
  it('should add styles', () => {
    expect(
      render(
        <VisuallyHidden>
          <span>Foo</span>
        </VisuallyHidden>
      ).asFragment()
    ).toMatchSnapshot('Without child styles')

    expect(
      render(
        <VisuallyHidden>
          <span style={{textAlign: 'center'}}>Foo</span>
        </VisuallyHidden>
      ).asFragment()
    ).toMatchSnapshot('With child styles')
  })
})
