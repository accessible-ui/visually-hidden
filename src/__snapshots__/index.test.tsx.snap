// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<VisuallyHidden> should add styles: With child styles 1`] = `
<DocumentFragment>
  <span
    style="border: 0px; height: 1px; width: 1px; margin: -1px; padding: 0px; overflow: hidden; position: absolute; text-align: center;"
  >
    Foo
  </span>
</DocumentFragment>
`;

exports[`<VisuallyHidden> should add styles: Without child styles 1`] = `
<DocumentFragment>
  <span
    style="border: 0px; height: 1px; width: 1px; margin: -1px; padding: 0px; overflow: hidden; position: absolute;"
  >
    Foo
  </span>
</DocumentFragment>
`;
